app/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -7,6 +7,7 @@
 from nimbus.pages.experimenter.summary import SummaryPage
 from nimbus.pages.remote_settings.dashboard import Dashboard
 from nimbus.pages.remote_settings.login import Login
+from nimbus.models.base_dataclass import BaseExperimentApplications
```

Comment:
@jaredlockhart looks like test failed bcause of import order 

ERROR: /app/tests/integration/nimbus/test_remote_settings.py Imports are incorrectly sorted and/or formatted.